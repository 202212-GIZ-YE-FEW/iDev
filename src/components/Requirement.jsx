export default function Requirement(props) {
    const { title, subtitle } = props;
    return (
        <div className='flex flex-col md:flex-row mb-5'>
            <div className='w-full'>
                <span className='font-semibold text-xl uppercase'>{title}</span>
                <p className='font-normal text-md lg:text-xl text-gray capitalize'>
                    {subtitle}
                </p>
            </div>
        </div>
    );
}
