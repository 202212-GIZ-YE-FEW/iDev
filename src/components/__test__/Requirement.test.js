import renderer from "react-test-renderer";

import Requirement from "../Requirement";
it("renders correctly", () => {
    const tree = renderer.create(<Requirement />).toJSON();
    expect(tree).toMatchSnapshot();
});
