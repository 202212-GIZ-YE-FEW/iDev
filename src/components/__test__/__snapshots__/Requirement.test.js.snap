// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="flex flex-col md:flex-row mb-5"
>
  <div
    className="w-full md:w-2/3"
  >
    <span
      className="font-semibold text-xl uppercase"
    />
    <p
      className="font-normal text-md lg:text-xl text-gray capitalize"
    />
  </div>
</div>
`;
